app/models/hydrus/generic_object_stuff.rb

```diff
@@ -6,7 +6,6 @@ module Hydrus::GenericObjectStuff
     include Hydrus::Validatable
     include Hydrus::Processable
     include Hydrus::Contentable
-    include Hydrus::WorkflowDsExtension
```

Comment:
I removed this module entirely since its methods weren't called anywhere other than specs.